### PR TITLE
final clean up!

### PR DESCRIPTION
**Pull request by liruochen1998**
… also run prettier on all files to make sure the styles are consistent

### CODE REVIEW

**Review by Jaega — DISMISSED** [review]
Thanks for the hard work!

File: README.md [hunk 1]
```diff
@@ -87,13 +87,18 @@ GPLv3
...
 
 
 
-## others
+## MISC. 
+### Stuctures of Code
+The entry point is `App.js` in  `./src`. Anything else can be found in `./src/components/*`. All the components follow the naming convention that inside `./src/components/MyComponent/`, we can find 3 files for most components (except Map) `MyComponent.js`, which is the main JS file, `MyComponent.css`, which is the css file, and `MyComponent.test.js`, which is the test file.    
+
+### Layout of Web UI
+The Layout of the Web UI can be changed by modifying the CSS file of each components. The CSS files are located inside the folder of each components.
```
Comment (carolinepar) [inline README.md]: So our elements are in flex boxes grouped by class so I'm not sure just editing CSS is enough... ie would need to edit className as well.  thoughts?

File: src/App.test.js [hunk 2]
```diff
@@ -14,39 +13,39 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
+  organizedObject["Alabama"] = {
+    deaths: {
+      "2/4/2020": 0,
+      "2/5/2020": 1,
+    },
+  };
+});
+test("renders App", () => {
   const app = render(<App />);
   //expect(app).toBeInTheDocument();
```
Comment (carolinepar) [inline src/App.test.js]: can we pull this old comment out?

File: src/components/DataDisplay/DataDisplay.test.js [hunk 3]
```diff
@@ -2,45 +2,41 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
```
Comment (carolinepar) [inline src/components/DataDisplay/DataDisplay.test.js]: should these be double quotes?

File: src/components/DataPanels/DataPanels.test.js [hunk 4]
```diff
@@ -16,104 +19,97 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
-    //expect(screen.getByTestId("DataDisplay")).toHaveTextContent(dateKey);
-    expect(screen.getByTestId("DataDisplay")).toHaveTextContent(data.toString());
+  const radio = screen.getByDisplayValue(buttonName);
+  const state = "Alabama";
+  const dateKey = "2/4/2020";
+  fireEvent.change(radio, { target: { value: buttonName } });
+  const data = getData(state, dateKey, buttonName);
+  expect(screen.getByTestId("DataDisplay")).toHaveTextContent(state);
+  expect(screen.getByTestId("DataDisplay")).toHaveTextContent(displayName);
+  //expect(screen.getByTestId("DataDisplay")).toHaveTextContent(dateKey);
```
Comment (carolinepar) [inline src/components/DataPanels/DataPanels.test.js]: can we delete this old comment too?

File: src/components/DataPanels/DataPanels.test.js [hunk 5]
```diff
@@ -16,104 +19,97 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
-	const data = getData(state, displayDate, 'deaths'); 
-	console.log("data: " + data);
-
-    expect(screen.getByTestId("DataDisplay")).toHaveTextContent(state);
-    expect(screen.getByTestId("DataDisplay")).toHaveTextContent(displayDate);
-    expect(screen.getByTestId("DataDisplay")).toHaveTextContent(data.toString());
-
-	
-}
+  //const radio = screen.getByDisplayValue(buttonName);
```
Comment (carolinepar) [inline src/components/DataPanels/DataPanels.test.js]: delete comment here and below

File: src/components/DataPanels/DataPanels.test.js [hunk 6]
```diff
@@ -16,104 +19,97 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
-          let dataForDate = dataForType[date];
-		  return dataForDate;
-        }
+  if (organizedObject) {
+    let dataForState = organizedObject[state];
+    if (dataForState) {
+      let destylizedType = type.replace(" ", "_");
+      let dataForType = dataForState[destylizedType];
+      if (dataForType) {
+        console.log(date);
```
Comment (carolinepar) [inline src/components/DataPanels/DataPanels.test.js]: delete console.log

File: src/components/Header/Header.js [hunk 7]
```diff
@@ -1,85 +1,89 @@
...
-      names = ['user1', 'user2', 'user3', 'user4'];
+      names = ["user1", "user2", "user3", "user4"];
     }
     names.sort(); // hack for everyone having the same order
     this.setState({
       users: names,
-      numOfUser: names.length
+      numOfUser: names.length,
     });
     console.log(names);
```
Comment (carolinepar) [inline src/components/Header/Header.js]: delete console.log

File: src/components/Header/Header.js [hunk 8]
```diff
@@ -1,85 +1,89 @@
...
-    })
+      currNum: currNum,
+    });
     if (!this.props.testEnv && window.TogetherJS.running) {
       window.TogetherJS.send({
-        type: 'turnUpdate',
-        currNum: currNum
-      })
+        type: "turnUpdate",
+        currNum: currNum,
```
Comment (carolinepar) [inline src/components/Header/Header.js:50]: delete this comma

File: src/components/Header/Header.js [hunk 9]
```diff
@@ -1,85 +1,89 @@
...
         numOfUser: msg.numOfUser,
       });
     });
-    window.TogetherJS.hub.on('turnUpdate', msg => {
+    window.TogetherJS.hub.on("turnUpdate", (msg) => {
       if (!msg.sameUrl) return;
-      this.setState( {
-        currNum: msg.currNum
+      this.setState({
+        currNum: msg.currNum,
```
Comment (carolinepar) [inline src/components/Header/Header.js:68]: delete this comma

File: src/components/LogTable/LogTable.js [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+    this.setState({
+      currItem: e.target.value,
+    });
+    this.props.handleFormChange(e);
+  }
+
+  handleSubmit() {
+    this.props.handleSubmit();
+    this.setState({
+      currItem: undefined,
```
Comment (carolinepar) [inline src/components/LogTable/LogTable.js:27]: delete comma

File: src/components/LogTableContainer/LogTableContainer.js [hunk 11]
```diff
@@ -1,179 +1,182 @@
...
+        window.TogetherJS.send({
+          type: "logTableUpdate",
+          log: currList,
+        });
+      }
+    }
   }
 
   undo() {
     // undo the previous log, do we need to switch user then?
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js]: address this comment and delete

File: src/components/LogTableContainer/LogTableContainer.js [hunk 12]
```diff
@@ -1,179 +1,182 @@
...
+    var info = infoList[this.state.activeIndex];
     var currList = info.items;
     currList.pop();
-		infoList[this.state.activeIndex].items = currList;
-		this.setState({
-			infos: infoList
-		});
+    infoList[this.state.activeIndex].items = currList;
+    this.setState({
+      infos: infoList,
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js:61]: delete comma

File: src/components/LogTableContainer/LogTableContainer.js [hunk 13]
```diff
@@ -1,179 +1,182 @@
...
+    });
 
     console.log(currList);
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js]: delete console log

File: src/components/LogTableContainer/LogTableContainer.js [hunk 14]
```diff
@@ -1,179 +1,182 @@
...
-			})
-		}
-		this.setState({
-			infos: infoList
-		});
+    var infoList = [];
+    for (let i = 0; i < numOfLogs; i++) {
+      infoList.push({
+        items: [],
+        currItem: undefined,
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js:90]: delete comma

File: src/components/LogTableContainer/LogTableContainer.js [hunk 15]
```diff
@@ -1,179 +1,182 @@
...
+      });
+    }
+    this.setState({
+      infos: infoList,
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js:94]: delete comma

File: src/components/LogTableContainer/LogTableContainer.js [hunk 16]
```diff
@@ -1,179 +1,182 @@
...
+    window.TogetherJS.hub.on("activeIndexUpdate", (msg) => {
       if (!msg.sameUrl) return;
       this.setState({ activeIndex: msg.index });
     });
   }
 
-	goToPrevTable() {
-		console.log('prev');
+  goToPrevTable() {
+    console.log("prev");
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js]: delete

File: src/components/LogTableContainer/LogTableContainer.js [hunk 17]
```diff
@@ -1,179 +1,182 @@
...
-		if(index < 1) {
+    if (index < 1) {
       index = length - 1;
     } else {
       index--;
     }
-		console.log('index: ' + index);
-		this.setState({
-      activeIndex: index
+    console.log("index: " + index);
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js]: delete

File: src/components/LogTableContainer/LogTableContainer.js [hunk 18]
```diff
@@ -1,179 +1,182 @@
...
+    this.setState({
+      activeIndex: index,
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js:115]: comma

File: src/components/LogTableContainer/LogTableContainer.js [hunk 19]
```diff
@@ -1,179 +1,182 @@
...
     });
     // together JS running update
     if (!this.props.testEnv && window.TogetherJS.running) {
       window.TogetherJS.send({
-        type: 'activeIndexUpdate',
-        index: index
+        type: "activeIndexUpdate",
+        index: index,
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js:127]: comma

File: src/components/LogTableContainer/LogTableContainer.js [hunk 20]
```diff
@@ -1,179 +1,182 @@
...
-      index = 0
+    if (index === length - 1) {
+      index = 0;
     } else {
       index++;
     }
-		console.log('index: ' + index);
-		this.setState({
-      activeIndex: index
+    console.log("index: " + index);
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js]: delete

File: src/components/LogTableContainer/LogTableContainer.js [hunk 21]
```diff
@@ -1,179 +1,182 @@
...
+    this.setState({
+      activeIndex: index,
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js:143]: comma

File: src/components/LogTableContainer/LogTableContainer.js [hunk 22]
```diff
@@ -1,179 +1,182 @@
...
-						</Button>
-						<Button onClick={() => this.goToNextTable()}>
-							<i className="caret right icon"></i> 
-						</Button>
-					</Button.Group>
-				</Segment>
-			</div>
-    )
+    console.log("infos");
+    console.log(this.state.infos);
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js]: delete console logs

File: src/components/LogTableContainer/LogTableContainer.js [hunk 23]
```diff
@@ -1,179 +1,182 @@
...
+    this.setState({
+      activeIndex: index,
     });
     // together JS running update
     if (!this.props.testEnv && window.TogetherJS.running) {
       window.TogetherJS.send({
-        type: 'activeIndexUpdate',
-        index: index
+        type: "activeIndexUpdate",
+        index: index,
```
Comment (carolinepar) [inline src/components/LogTableContainer/LogTableContainer.js:149]: comma

File: src/components/Map/Map.js [hunk 24]
```diff
@@ -1,39 +1,45 @@
...
+  let dataLayerTypes = {};
   let headings = getHeadings();
-  headings.forEach(curr => {
-      const enumValue = curr;
-      dataLayerTypes = {...dataLayerTypes, [enumValue]: {name: curr, layerId: curr}};
+  headings.forEach((curr) => {
+    const enumValue = curr;
+    dataLayerTypes = {
+      ...dataLayerTypes,
+      [enumValue]: { name: curr, layerId: curr },
```
Comment (carolinepar) [inline src/components/Map/Map.js:28]: comma

File: src/components/Map/Map.js [hunk 25]
```diff
@@ -1,39 +1,45 @@
...
+    };
   });
-  enumDataLayerType = {...dataLayerTypes}; //TODO: replace this weith dataLayerTypes everywhere since it's no longer enum
-}
+  enumDataLayerType = { ...dataLayerTypes }; //TODO: replace this weith dataLayerTypes everywhere since it's no longer enum
```
Comment (carolinepar) [inline src/components/Map/Map.js]: delete comment

File: src/components/Map/Map.js [hunk 26]
```diff
@@ -1,39 +1,45 @@
...
 
 class Map extends React.Component {
-
   constructor(props) {
     super(props);
     this.state = {
       lat: 35.9,
       lng: -79.1,
-      zoom: 3
+      zoom: 3,
```
Comment (carolinepar) [inline src/components/Map/Map.js:42]: comma

File: src/components/Map/Map.js [hunk 27]
```diff
@@ -43,72 +49,74 @@ class Map extends React.Component {
     // restrict user inside this region of the map
     this.bounds = [
       [-179, -5.91619], // Southwest coordinates
-      [-20.96466, 75.3577635769] // Northeast coordinates
+      [-20.96466, 75.3577635769], // Northeast coordinates
```
Comment (carolinepar) [inline src/components/Map/Map.js:52]: comma

File: src/components/Map/Map.js [hunk 28]
```diff
@@ -43,72 +49,74 @@ class Map extends React.Component {
...
     this.currentDataLayer = null;
     this.finishLoadingMapStyle = false;
   }
 
-  initDataLayer(dataId, layerType, visible) { 
+  initDataLayer(dataId, layerType, visible) {
     const layerId = layerType.layerId;
-    const layerColor = layerColors['mapExample'];
+    const layerColor = layerColors["mapExample"];
     // const layerColor = layerColors[layerType.name];
```
Comment (carolinepar) [inline src/components/Map/Map.js:63]: delete this comment

File: src/components/Map/Map.js [hunk 29]
```diff
@@ -43,72 +49,74 @@ class Map extends React.Component {
...
+            ["boolean", ["feature-state", "hover"], false],
+            ["boolean", ["feature-state", "select"], false],
+            false,
           ],
           0.85,
-          0.5
-        ]
-      }
-
+          0.5,
```
Comment (carolinepar) [inline src/components/Map/Map.js:106]: comma

File: src/components/Map/Map.js [hunk 30]
```diff
@@ -43,72 +49,74 @@ class Map extends React.Component {
...
+        ],
+      },
     });
-    this.map.setFilter(layerId, ['==', ['string', ['get', 'date']], mapIntToDate(this.props.selectedDate)]);
+    this.map.setFilter(layerId, [
+      "==",
+      ["string", ["get", "date"]],
+      mapIntToDate(this.props.selectedDate),
```
Comment (carolinepar) [inline src/components/Map/Map.js:113]: comma

File: src/components/Map/Map.js [hunk 31]
```diff
@@ -43,72 +49,74 @@ class Map extends React.Component {
...
+    ]);
     console.log(layerId);
```
Comment (carolinepar) [inline src/components/Map/Map.js]: console log

File: src/components/Map/Map.js [hunk 32]
```diff
@@ -156,76 +167,86 @@ class Map extends React.Component {
...
 
   componentDidMount() {
     this.map = new mapboxgl.Map({
       container: this.mapContainer,
-      style: 'mapbox://styles/mapbox/streets-v11',
+      style: "mapbox://styles/mapbox/streets-v11",
       center: [this.state.lng, this.state.lat],
       zoom: this.state.zoom,
-      maxBounds: this.bounds
+      maxBounds: this.bounds,
```
Comment (carolinepar) [inline src/components/Map/Map.js:197]: comma

File: src/components/Map/Map.js [hunk 33]
```diff
@@ -156,76 +167,86 @@ class Map extends React.Component {
...
         layout: {},
         paint: {
-          'line-color': '#FFFFFF',
-          'line-width': 2,
-          'line-dasharray': [3, 3]
-        }
+          "line-color": "#FFFFFF",
+          "line-width": 2,
+          "line-dasharray": [3, 3],
+        },
```
Comment (carolinepar) [inline src/components/Map/Map.js:223]: comma

File: src/components/Map/Map.js [hunk 34]
```diff
@@ -156,76 +167,86 @@ class Map extends React.Component {
...
-
-
   }
   componentDidUpdate(prevProps) {
-    if ((prevProps.selectedDate !== this.props.selectedDate) && this.finishLoadingMapStyle) {
+    if (
+      prevProps.selectedDate !== this.props.selectedDate &&
+      this.finishLoadingMapStyle
+    ) {
       // console.log(this.props.selectedDate);
```
Comment (carolinepar) [inline src/components/Map/Map.js]: delete

File: src/components/Map/layerColors.js [hunk 35]
```diff
@@ -1,43 +1,43 @@
...
+  tests: {
+    0: "#2df2f2",
+    10: "#22daee",
+    20: "#1ec1e6",
+    50: "#20a2da",
+    100: "#2384ca",
+    200: "#2560b8",
+    500: "#2951a2",
+    1000: "#2f258b",
+  },
```
Comment (carolinepar) [inline src/components/Map/layerColors.js:42]: comma

File: src/components/Map/tokens.js [hunk 36]
```diff
@@ -1,3 +1,4 @@
 // TODO: tokens for testing only
```
Comment (carolinepar) [inline src/components/Map/tokens.js:1]: fix and delete comment

File: src/components/MapAndDataContainer/MapAndDataContainer.js [hunk 37]
```diff
@@ -26,53 +32,54 @@ class MapAndDataContainer extends React.Component {
...
+      this.setState({ [type]: [value] }, callback);
+    }
+    this.setState({ [type]: [value] });
+    if (!this.props.testEnv) {
+      // sync togetherJS
+      if (window.TogetherJS.running) {
+        window.TogetherJS.send({
+          type: "dataUpdate",
+          dataType: type,
+          dataValue: value,
```
Comment (carolinepar) [inline src/components/MapAndDataContainer/MapAndDataContainer.js:45]: comma

File: src/components/MapAndDataContainer/MapAndDataContainer.js [hunk 38]
```diff
@@ -92,43 +99,40 @@ class MapAndDataContainer extends React.Component {
...
-    	  this.setState({[type]: [value]});
-    	});
-	}
+    if (!this.props.testEnv) {
+      // register together Sync
+      window.TogetherJS.hub.on("dataUpdate", (msg) => {
+        if (!msg.sameUrl) return;
+        const type = msg.dataType;
+        const value = msg.dataValue;
+        // can potentially call setData, but I'm afraid that it will run into a circle
```
Comment (carolinepar) [inline src/components/MapAndDataContainer/MapAndDataContainer.js]: delete

File: src/components/QaPanel/QaPanel.css [hunk 39]
```diff
@@ -0,0 +1,14 @@
+.qaPanel {
+}
+
+.googleForm {
+  left: 250px;
+  position: absolute;
+  z-index: 1;
+  bottom: 20px;
+  /* border: black solid 2px; */
```
Comment (carolinepar) [inline src/components/QaPanel/QaPanel.css]: delete comments

File: src/components/QaPanel/QaPanel.js [hunk 40]
```diff
@@ -0,0 +1,26 @@
...
+        marginHeight="0"
+        marginWidth="0"
+      >
+        Loading…
+      </iframe>
+    );
+  }
+}
+
+//in the future, use className freebirdFormviewerComponentsQuestionBaseDescription and get the first character in the string to get the number of allowed items to log, then post message it up.
```
Comment (carolinepar) [inline src/components/QaPanel/QaPanel.js]: delete

File: src/components/RadioButtonPanel/RadioButtonPanel.js [hunk 41]
```diff
@@ -1,61 +1,73 @@
...
-        this.headings = getStyledHeadings();
-        this.state={
-            checkedButton: getDefaultHeading()
-        }
-        this.handleChange = this.handleChange.bind(this);
-        
-    }
+  handleChange(event) {
+    this.setState({ checkedButton: event.target.value });
+    // this.props.setData('selectedDataType', event.target.value);
```
Comment (carolinepar) [inline src/components/RadioButtonPanel/RadioButtonPanel.js]: delete

File: src/components/RadioButtonPanel/RadioButtonPanel.js [hunk 42]
```diff
@@ -1,61 +1,73 @@
...
-        this.setState({checkedButton: event.target.value});
-        // this.props.setData('selectedDataType', event.target.value);
-        this.props.changeDataType(event.target.value);
-    }
-    
-    // call window.pageComponent.getCheckedButton() to get the checked button or:: window.radioButtonComponent.getCheckedButton()
-    getCheckedButton(){
-        return this.state.checkedButton;
-    }
+  // call window.pageComponent.getCheckedButton() to get the checked button or:: window.radioButtonComponent.getCheckedButton()
```
Comment (carolinepar) [inline src/components/RadioButtonPanel/RadioButtonPanel.js]: delete

File: src/components/RadioButtonPanel/RadioButtonPanel.test.js [hunk 43]
```diff
@@ -15,70 +14,65 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
+    dataObject.push({
+      date: "2020-02-04",
+      deaths: "0",
+      id: "1089",
+      infections: "0",
+      location: "Alabama",
+      tests: "0",
+      total_deaths: "0",
+      total_infections: "0",
+      total_tests: "0",
```
Comment (carolinepar) [inline src/components/RadioButtonPanel/RadioButtonPanel.test.js:30]: comma

File: src/components/RadioButtonPanel/RadioButtonPanel.test.js [hunk 44]
```diff
@@ -15,70 +14,65 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
+    });
+    organizedObject["Alabama"] = {
+      deaths: {
+        "2/4/2020": 0,
```
Comment (carolinepar) [inline src/components/RadioButtonPanel/RadioButtonPanel.test.js:34]: comma

File: src/components/RadioButtonPanel/RadioButtonPanel.test.js [hunk 45]
```diff
@@ -15,70 +14,65 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
+      },
```
Comment (carolinepar) [inline src/components/RadioButtonPanel/RadioButtonPanel.test.js:35]: comma

File: src/components/RadioButtonPanel/RadioButtonPanel.test.js [hunk 46]
```diff
@@ -15,70 +14,65 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
-//	render(<MapAndDataContainer testEnv='true'/>);
-//	const testText = 'deaths';
-//	testRadioToDataDisplay(testText, testText);
-//  });
+  test("it should mount", () => {
+    render(<MapAndDataContainer testEnv="true" />);
+    const radioButton = screen.getByTestId("RadioButton");
+    expect(radioButton).toBeInTheDocument();
+  });
+  //  test('radio button should trigger the right data', () => {
```
Comment (carolinepar) [inline src/components/RadioButtonPanel/RadioButtonPanel.test.js]: delete

File: src/components/RadioButtonPanel/RadioButtonPanel.test.js [hunk 47]
```diff
@@ -15,70 +14,65 @@ jest.mock('mapbox-gl/dist/mapbox-gl', () => ({
...
-        }
+  if (organizedObject) {
+    let dataForState = organizedObject[state];
+    if (dataForState) {
+      let destylizedType = type.replace(" ", "_");
+      let dataForType = dataForState[destylizedType];
+      if (dataForType) {
+        console.log(date);
+        let dataForDate = dataForType[date];
+        return dataForDate;
```
Comment (carolinepar) [inline src/components/RadioButtonPanel/RadioButtonPanel.test.js]: comma

File: src/components/Slider/Slider.js [hunk 48]
```diff
@@ -15,32 +14,44 @@ class Slider extends React.Component {
...
-    let value = parseInt(document.getElementById('dateSlider').value) + 1;
-    this.handleChange({target:{value}});
+  handleRightArrowClick() {
+    let value = parseInt(document.getElementById("dateSlider").value) + 1;
+    this.handleChange({ target: { value } });
   }
 
-  render(){
+  render() {
     console.log(this.props.dateRange);
```
Comment (carolinepar) [inline src/components/Slider/Slider.js]: delete

File: src/components/Slider/Slider.test.js [hunk 49]
```diff
@@ -18,18 +17,20 @@ describe('<Slider />', () => {
       tests: "0",
       total_deaths: "0",
       total_infections: "0",
-      total_tests: "0"
+      total_tests: "0",
```
Comment (carolinepar) [inline src/components/Slider/Slider.test.js:20]: comma

File: src/index.js [hunk 50]
```diff
@@ -1,24 +1,23 @@
...
       <App />
     </React.StrictMode>,
-    document.getElementById('root')
+    document.getElementById("root")
   );
-}
+};
 newPromise.then(renderApp);
 
 // If you want your app to work offline and load faster, you can change
```
Comment (carolinepar) [inline src/index.js:23]: delete comment

File: src/utils/data.js [hunk 51]
```diff
@@ -1,141 +1,152 @@
...
+};
 
-export const getMinDate = function(){
-    //TODO: store this in the object
-    return getDateObject().reduce(function(prev, curr) {
-        return prev.date < curr.date ? prev : curr;
-    });
-}
+export const getMinDate = function () {
+  //TODO: store this in the object
```
Comment (carolinepar) [inline src/utils/data.js]: delete

File: src/utils/data.js [hunk 52]
```diff
@@ -1,141 +1,152 @@
...
+};
 
-export const getMaxDate = function(){
-    //TODO: store this in the object
-    return getDateObject().reduce(function(prev, curr) {
-        return prev.date > curr.date ? prev : curr;
-    });
-}
+export const getMaxDate = function () {
+  //TODO: store this in the object
```
Comment (carolinepar) [inline src/utils/data.js]: delete

File: src/utils/data.js [hunk 53]
```diff
@@ -1,141 +1,152 @@
...
+    const infections = Math.floor(parseFloat(entry.infections));
+    const totalInfections = Math.floor(parseFloat(entry.total_infections));
+    if (!organizedObject[stateName]) {
+      organizedObject[stateName] = {
+        deaths: {},
+        total_deaths: {},
+        tests: {},
+        total_tests: {},
+        infections: {},
+        total_infections: {},
```
Comment (carolinepar) [inline src/utils/data.js:118]: comma

**Review by carolinepar — COMMENTED** [review]
Thank you for doing this!!!! Sorry for so many comments lol, just deleting comments, console.logs and a bunch of free floating commas at the end of objects

**Review by carolinepar — APPROVED** [review]

**Review by Jaega — APPROVED** [review]

**Pull request merged**